influxdb/_dataframe_client.py

```diff
@@ -131,25 +132,26 @@ def write_points(self,
                 protocol=protocol)
             return True
 
-    def query(self, query, chunked=False, database=None):
+    def query(self, query, dropna=True, **kwargs):
         """
         Quering data into a DataFrame.
 
-        :param chunked: [Optional, default=False] True if the data shall be
-            retrieved in chunks, False otherwise.
+        :param query: the actual query string
+        :param dropna: drop columns where all values are missing
+        :param **kwargs: additional parameters for ``InfluxDBClient.query``
 
         """
-        results = super(DataFrameClient, self).query(query, database=database)
+        results = super(DataFrameClient, self).query(query, **kwargs)
```

Comment:
Thanks for the feedback. 

As you said, "maintainers to set guidelines on things like this", so I will just follow what the maintainers decide then :)

I hadn't really thought about static analyser tools and about the relation between `kwargs` and private/public functions. I am glad we had this little discussion.

Have you taken a chance to look at my latest commits? Please let me know if you think the PR is mergeable now. If not, please let me know what else I should change.